project-koku/koku
# Preventing ingress report spamming

## Jira Ticket

[COST-3844](https://issues.redhat.com/browse/COST-3844)

## Description

This change will prevent you posting ingress reports for a currently processing billing month until the current processing is complete.

## Testing

1. Checkout Branch
2. Restart Koku
3. Add a storage only source
4. Post ingress reports via /api/cost-management/v1/ingress/reports/
5. Immediately post the same report/billing period
6. Should raise an error

## Notes

...
